jf_agent/main.py

```diff
@@ -47,7 +49,7 @@
 
 logger = logging.getLogger(__name__)
 
-JELLYFISH_API_BASE = 'https://jellyfish.co'
+JELLYFISH_API_BASE = 'http://localhost:8000'
```

Comment:
I think you don't want to check this in 😁

Reply:
🤦‍♀️ whoops